src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -47,7 +47,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn),0419,0649</NoWarn>
+    <NoWarn>$(NoWarn),0419,0649,CA2249,CA1830</NoWarn>
```

Comment:
Why are these disabled for corelib?

Reply:
CA2249 is "Prefer string.Contains over string.IndexOf" and CA1830 is "Prefer typed StringBuilder overloads". Both these APIs are defined in corelib right? So they'd trigger the analyzer but there's nothing to do other than suppress them. I could make the suppression local if you'd prefer that